uniffi_macros/src/export/metadata/convert.rs

```diff
@@ -21,6 +21,11 @@ pub(super) fn fn_param_metadata(
                         syn::Pat::Ident(pat_id) => pat_id.ident.to_string(),
                         _ => unimplemented!(),
                     };
+
+                    if name == "self" {
+                        return None;
+                    }
```

Comment:
And a comment here about why we don't output any metadata since there's an implicit `self`.